packages/core/src/textInput/styles/LabelInner.ts

```diff
@@ -8,7 +8,7 @@ const LabelInner = styled.label(({ theme: { color, transitionDuration } }) => ({
   lineHeight: rem(12),
   color: color.trunks[100],
   top: rem(11),
-  zIndex: 1,
+  zIndex: 3,
```

Comment:
consider `pointer-events: none;`

Reply:
does not solve the z-index problem. The label is still under the input

Reply:
@alexisdevtailor put Label after Input component then.